cloud-controller-manager/cmd/digitalocean-cloud-controller-manager/main.go

```diff
@@ -72,3 +79,13 @@ func doInitializer(cfg *config.CompletedConfig) cloudprovider.Interface {
 
 	return cloud
 }
+
+func startWebhookServer() {
+	// default server running at port 9443, looking for tls.crt, tls.key in /tmp/k8s-webhook-server/serving-certs
+	server := webhook.Server{}
+
+	server.Register("/validate-doks-lb-service", &webhook.Admission{Handler: &dokswebhooks.DOKSLBServiceValidator{Client: c, Log: ll}})
+	if err := server.StartStandalone(ctrl.SetupSignalHandler(), scheme); err != nil {
+		os.Exit(1)
```

Comment:
It's recommended to do this only from main, helpers can return an error to main.